src/components/fullscreen/fullscreen.tsx

```diff
@@ -92,6 +100,10 @@ class Fullscreen extends Component<
    */
   handleKeydown(event: KeyboardEvent): void {
     switch (event.keyCode) {
+      case KeyMap.ENTER:
+        if (!event.repeat) {
```

Comment:
did you try debouncing the event handler ?

Reply:
as i know debounce is used for delay, i don't want to wait, i want the action not to happen at all.